android/tangram/src/com/mapzen/tangram/HttpHandler.java

```diff
@@ -49,8 +61,7 @@ public void onCancel(String url) {
      */
     public boolean setCache(File directory, long maxSize) {
         Cache okTileCache = new Cache(directory, maxSize);
-        okClient.setCache(okTileCache);
-
+        okClient = okClientBuilder.cache(okTileCache).build();
```

Comment:
Is there a reason you are building a new okClient here, instead of setting the cache on the already built okClient (in the HttpHandler constructor)? If this is an API change I am not able to find it.